{
  "build": "py311haa95532_0",
  "build_number": 0,
  "channel": "https://repo.anaconda.com/pkgs/main/win-64",
  "constrains": [],
  "depends": [
    "python >=3.11,<3.12.0a0"
  ],
  "extracted_package_dir": "C:\\Users\\sangw\\miniconda3\\pkgs\\exceptiongroup-1.2.0-py311haa95532_0",
  "files": [
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/INSTALLER",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/LICENSE",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/METADATA",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/RECORD",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/REQUESTED",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/WHEEL",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/direct_url.json",
    "Lib/site-packages/exceptiongroup/__init__.py",
    "Lib/site-packages/exceptiongroup/__pycache__/__init__.cpython-311.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_catch.cpython-311.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-311.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_formatting.cpython-311.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_suppress.cpython-311.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_version.cpython-311.pyc",
    "Lib/site-packages/exceptiongroup/_catch.py",
    "Lib/site-packages/exceptiongroup/_exceptions.py",
    "Lib/site-packages/exceptiongroup/_formatting.py",
    "Lib/site-packages/exceptiongroup/_suppress.py",
    "Lib/site-packages/exceptiongroup/_version.py",
    "Lib/site-packages/exceptiongroup/py.typed",
    ".nonadmin"
  ],
  "fn": "exceptiongroup-1.2.0-py311haa95532_0.conda",
  "license": "MIT",
  "link": {
    "source": "C:\\Users\\sangw\\miniconda3\\pkgs\\exceptiongroup-1.2.0-py311haa95532_0",
    "type": 1
  },
  "md5": "a7a3462e4237c21f213fb13eec5b1d48",
  "name": "exceptiongroup",
  "package_tarball_full_path": "C:\\Users\\sangw\\miniconda3\\pkgs\\exceptiongroup-1.2.0-py311haa95532_0.conda",
  "paths_data": {
    "paths": [
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/INSTALLER",
        "path_type": "hardlink",
        "sha256": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "sha256_in_prefix": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "size_in_bytes": 5
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/LICENSE",
        "path_type": "hardlink",
        "sha256": "6e5070d765031e0ad403a1cbf90ae6d19a0228f802e320b7acff460aa72ed47c",
        "sha256_in_prefix": "6e5070d765031e0ad403a1cbf90ae6d19a0228f802e320b7acff460aa72ed47c",
        "size_in_bytes": 3704
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/METADATA",
        "path_type": "hardlink",
        "sha256": "1d998cf5f14e54cf261360d854a049606682f73813b6a22d1b1d7bd51f17ba2e",
        "sha256_in_prefix": "1d998cf5f14e54cf261360d854a049606682f73813b6a22d1b1d7bd51f17ba2e",
        "size_in_bytes": 6603
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/RECORD",
        "path_type": "hardlink",
        "sha256": "fdbb8d534046b8b2945b1a5061de492771ac197d6666a1a04afabb0312ca0d6f",
        "sha256_in_prefix": "fdbb8d534046b8b2945b1a5061de492771ac197d6666a1a04afabb0312ca0d6f",
        "size_in_bytes": 1541
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/REQUESTED",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/WHEEL",
        "path_type": "hardlink",
        "sha256": "1196c6921ec87b83e865f450f08d19b8ff5592537f4ef719e83484e546abe33e",
        "sha256_in_prefix": "1196c6921ec87b83e865f450f08d19b8ff5592537f4ef719e83484e546abe33e",
        "size_in_bytes": 81
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/direct_url.json",
        "path_type": "hardlink",
        "sha256": "a049c9b1baeaba827a6678f6686509f3c0d7a3ddb3cd462ca5146b0c83929755",
        "sha256_in_prefix": "a049c9b1baeaba827a6678f6686509f3c0d7a3ddb3cd462ca5146b0c83929755",
        "size_in_bytes": 94
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__init__.py",
        "path_type": "hardlink",
        "sha256": "ec31d2d210e4f9122cdc841cdd26d9541d3ed4c86248227d5e0bc4c849682fb3",
        "sha256_in_prefix": "ec31d2d210e4f9122cdc841cdd26d9541d3ed4c86248227d5e0bc4c849682fb3",
        "size_in_bytes": 1049
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/__init__.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "43f78404bd8932e9dfd7db9238919586a582ee01d90694c09514625648f37e52",
        "sha256_in_prefix": "43f78404bd8932e9dfd7db9238919586a582ee01d90694c09514625648f37e52",
        "size_in_bytes": 1241
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_catch.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "f00166b66c9fb90f5d6952869ccee974fc987261d3a60c2c78588ff71cdefba2",
        "sha256_in_prefix": "f00166b66c9fb90f5d6952869ccee974fc987261d3a60c2c78588ff71cdefba2",
        "size_in_bytes": 5565
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "764b0fabc8e3ad288203d8ef76f840111194b41b8a5446a2287f1d597d7bd9a2",
        "sha256_in_prefix": "764b0fabc8e3ad288203d8ef76f840111194b41b8a5446a2287f1d597d7bd9a2",
        "size_in_bytes": 15103
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_formatting.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "644c26ff9543dc5a37797a6c0c10f05aa014357a9f8feccb737bef8d6271ef12",
        "sha256_in_prefix": "644c26ff9543dc5a37797a6c0c10f05aa014357a9f8feccb737bef8d6271ef12",
        "size_in_bytes": 23151
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_suppress.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "cf8c93c8b42e7acdc267c79f7d06fa219f0e53184a998019b5d10ea9cb429384",
        "sha256_in_prefix": "cf8c93c8b42e7acdc267c79f7d06fa219f0e53184a998019b5d10ea9cb429384",
        "size_in_bytes": 1478
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_version.cpython-311.pyc",
        "path_type": "hardlink",
        "sha256": "e4e216719a0e62a26fc723eb12c5d49dc15b36bc7aa8fe624bea8676acf15de5",
        "sha256_in_prefix": "e4e216719a0e62a26fc723eb12c5d49dc15b36bc7aa8fe624bea8676acf15de5",
        "size_in_bytes": 263
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_catch.py",
        "path_type": "hardlink",
        "sha256": "926c74d4b904648e7ea66c61f404c43b8bf3a80b649fb9e357ee59f85d0aa7e2",
        "sha256_in_prefix": "926c74d4b904648e7ea66c61f404c43b8bf3a80b649fb9e357ee59f85d0aa7e2",
        "size_in_bytes": 4669
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_exceptions.py",
        "path_type": "hardlink",
        "sha256": "7fe58e5c7c3e9555800198c663c73e285c687c2bc37049785de3037aa7203328",
        "sha256_in_prefix": "7fe58e5c7c3e9555800198c663c73e285c687c2bc37049785de3037aa7203328",
        "size_in_bytes": 11010
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_formatting.py",
        "path_type": "hardlink",
        "sha256": "9ae98881193ad6d86fef03fa5b30a954db063e16191967df25e1b34278052f4b",
        "sha256_in_prefix": "9ae98881193ad6d86fef03fa5b30a954db063e16191967df25e1b34278052f4b",
        "size_in_bytes": 21005
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_suppress.py",
        "path_type": "hardlink",
        "sha256": "f3298ac7900570ed79178ecd7e7448f86f7c9a7e19057bb2b78aa22dde730f74",
        "sha256_in_prefix": "f3298ac7900570ed79178ecd7e7448f86f7c9a7e19057bb2b78aa22dde730f74",
        "size_in_bytes": 1338
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_version.py",
        "path_type": "hardlink",
        "sha256": "3d96f686ea42f34d7ccf213690b58eddd919a14ad655174f06325d55f04d7547",
        "sha256_in_prefix": "3d96f686ea42f34d7ccf213690b58eddd919a14ad655174f06325d55f04d7547",
        "size_in_bytes": 164
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/py.typed",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      }
    ],
    "paths_version": 1
  },
  "requested_spec": "exceptiongroup==1.2.0=py311haa95532_0",
  "sha256": "fe890657b844030c6e604dc0fb064e2c781484fcaae8291691f14a4b026d5a57",
  "size": 38362,
  "subdir": "win-64",
  "timestamp": 1706031560000,
  "url": "https://repo.anaconda.com/pkgs/main/win-64/exceptiongroup-1.2.0-py311haa95532_0.conda",
  "version": "1.2.0"
}